scripts/index_d_ts

```diff
@@ -1,3 +1,5 @@
+import { ChangeEventHandler } from 'react';
```

Comment:
Is this okay?

Reply:
probably better move inside `SendbirdUIKitGlobal`,

Reply:
actually, see comment below